src/tools/cephfs/cephfs-shell

```diff
@@ -1100,23 +1101,41 @@ sub-directories, files')
         """
         Disk Usage of a Directory
         """
-        if args.dirs == '':
-            args.dirs = cephfs.getcwd().decode('utf-8')
-        for dir_ in args.dirs:
-            if args.r:
-                for i in reversed(sorted(set(dirwalk(dir_)))):
-                    i = os.path.normpath(i)
+        dir_passed = args.dirs
+        if args.dirs is '':
+            dir_passed = cephfs.getcwd().decode('utf-8')
+            if len(dir_passed) > 1:
+                dir_passed = dir_passed.split()
+
+        for dir_ in dir_passed:
+                dir_trav = sorted(set(dirwalk(dir_)))
+                if args.dirs and not dir_trav:
+                    dir_trav.append(dir_)
+                else:
+                    dir_trav.append('.')
+
+                if args.r:
+                    dir_trav = reversed(dir_trav)
+
+                for i in dir_trav:
```

Comment:
We don't need this loop at all. We should just print the disk usage for each argument. There's no need to recurse.

Reply:
Update: I and Patrick talked about this part and decided that we'll keep this loop (i.e. we'll require recursion). The change is that du will not behave recursively, by default.

Reply:
The change has been made, resolving this conversation.